packages/CatalogService/index.ts

```diff
@@ -109,7 +110,8 @@ export class DCATResource extends RDFSResource {
         this.description = statement?.description.value;
       }
       if (statement?.contactEmail) {
-        this.contactEmail = statement?.contactEmail.value;
+        let email = statement?.contactEmail.value;
+        this.contactEmail = email.substring(email.lastIndexOf("/") + 1);
```

Comment:
**Observation**: 
 naively, I assume email is a url where the last uri segment is an email.

**Idea:**
If the uri is hard coded - perhaps you can search for more than "/"
e.g. `email.lastIndexOf('http://schema.org/email/')`